src/modules/general/apiLoadSettings.js

```diff
@@ -16,6 +17,8 @@ export const apiLoadGeneralSettings = async () => {
                         })
                 }
             })
+        } else {
+            navigateTo('/')
```

Comment:
I think it would be better to have this happening in a component, rather than here in the api function.